pokete.py

```diff
@@ -404,18 +421,22 @@ def attack(self, attac, enem):
             enem.oldhp = enem.hp
             self.oldhp = self.hp
             effectivity = (1.3 if enem.type.name in attac.type.effective
-                            else 0.5
-                            if enem.type.name in attac.type.ineffective
-                            else 1)
-            n_hp = round((self.atc * attac.factor / (enem.defense if enem.defense >= 1 else 1))*random.choices([0, 0.75, 1, 1.26], weights=[attac.miss_chance+self.miss_chance, 1, 1, 1], k=1)[0]*effectivity)
+                           else 0.5
+                           if enem.type.name in attac.type.ineffective
+                           else 1
+                           )
```

Comment:
Why does the bracket close here and not the lone before?

Reply:
Better readability, so you know by looking, when the nested if-else statements are done.

Reply:
But we could put it in the line before as well, if that makes you happy :)